<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTableWidget" name="leftTableWidget">
      <column>
       <property name="text">
        <string>Directories</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QTreeWidget" name="rightTreeWidget">
     <attribute name="headerMinimumSectionSize">
         <number>390</number>
     </attribute>
      <attribute name="headerDefaultAlignment">
       <enum>AlignLeft</enum>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Similar Files</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionAdd_Directory"/>
    <addaction name="actionScan_Directory"/>
    <addaction name="actionRemove_Directories_From_List"/>
    <addaction name="actionRemove_Files"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>directory parameters</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Directory"/>
   <addaction name="actionRemove_Directories_From_List"/>
   <addaction name="separator"/>
   <addaction name="actionRemove_Files"/>
  </widget>
  <widget class="QToolBar" name="paramsBar">
   <property name="windowTitle">
    <string>scan parameters</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <widget class="QCheckBox" name="recursive">
       <property name="text">
           <string>Subdirectories</string>
       </property>
   </widget>
   <widget class="QCheckBox" name="hidden">
       <property name="text">
           <string>Hidden Files</string>
       </property>
   </widget>
     <addaction name="actionScan_Directory"/>
  </widget>
  <action name="actionAdd_Directory">
   <property name="text">
    <string>&amp;Add Directory...</string>
   </property>
  </action>
  <action name="actionScan_Directory">
   <property name="text">
    <string>&amp;Scan Directory...</string>
   </property>
  </action>
  <action name="actionRemove_Files">
   <property name="text">
    <string>&amp;Remove Files...</string>
   </property>
  </action>
  <action name="actionRemove_Directories_From_List">
   <property name="text">
    <string>&amp;Remove Directories from List...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
